### DISCUSSION TITLE
[Video_demo.py] Can I load multiple feature extractor's weights?

### DISCUSSION THREAD

**Discussion by Deo-ahn** [ekosman/AnomalyDetectionCVPR2018-Pytorch#209] [Announcements]
Hi, thanks for nice work here. 

Now,  I have extracted features from multiple feature extractor models. 

And I trained them with one AD model(FC layer). 

But with your 'Video_demo.py', I can only load 1 feature extractor model weight. 

Do you think that i could modify the code to run with multiple feature extractor model?

I'm looking for the solutions to fix this but as far, i couldn't so. 
hope that you may could see if it is possible. 

Questions Summarize: 

1. Can I load multiple feature extractor model weights ?

2. Or IF I don't "have to" load the feature extractor weights, then 
How about the concatenated features to loaded. 

thank you for reading. 

Best regards.

**Comment by ekosman** [comment]
In order to help, I need to understand the new framework better. Could you maybe draw it?
